test/unit/assignmentDestructuring.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Expect, Test, TestCase } from "alsatian";
+import * as util from "../src/util";
+import { LuaTarget } from "../../src/Transpiler";
+
+export class AssignmentDestructuringTests {
+    
+    private readonly assignmentDestruturingTs = `
+    declare function myFunc(): [number, string];
```

Comment:
Indent line 8 and 9 please.

Reply:
Almost all test files have issues reported by `tslint`. I'm using VS Code and the extension seems to also apply the lint to the test files. I think `./tests/**/*.ts` should be ignored by `tslint`?

**tslint.json**
```js
{
/// ...
    "linterOptions": {
        "exclude": [
            "./test/**/*.ts"
        ]
    },
// ...
}
```

Reply:
Eventually they should adhere to tslint I think, but for now I am treating tslint as "suggestions" for test files (which is why it doesn't fail builds).

Reply:
Is it okay to disable `tslint` for `./test` for now (as demonstrated above)? I'm using VS Code and it's rather obtrusive to have the test files full of errors. Unless a path's severity can be forced to "suggestion"?

Reply:
As long as you don't push it you can change it however you want :D